src/components/article-body/body-components/table-row.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { ComponentFactory } from '../component-factory';
+import { ArticleNode } from '../../../interfaces/article-body-node';
+
+export const TableRow = ({
+    align,
+    children,
+}: {
+    align: any;
+    children: any;
+}) => (
+    <tbody>
+        <tr>
+            {children.map((column: ArticleNode, index: number) => (
+                <ComponentFactory
+                    // Pass the align array so we can apply alignment in TableCell
+                    nodeData={{ align, index, ...column }}
+                    key={index}
```

Comment:
Don't use the index as the key as it could cause issues in React https://medium.com/geekculture/reactjs-why-index-as-a-key-is-an-anti-pattern-4b9dc6ef0067